src/components/ExtractorResponseEditor/EntityPickerContainer.tsx

```diff
@@ -196,24 +195,23 @@ export default class EntityPickerContainer extends React.Component<Props, State>
     }
 
     render() {
```

Comment:
huh?

Reply:
Oops, was using it as reference to preserve functionality and forgot to delete it.